pkg/manager/config/init.go

```diff
@@ -159,3 +161,17 @@ func makeOverrideConfig() *viper.Viper {
 
 	return viper
 }
+
+// Shutdown gracefully stops the application
+func Shutdown(code int) {
+	// TODO: Each part of the manager needs to gracefully stop at some point
+	// for now, we just close the database.
+	err := database.Close()
+	if err != nil {
+		logger.Errorf("Error closing database: %s", err)
+		if code == 0 {
+			os.Exit(1)
+		}
+	}
+	os.Exit(code)
+}
```

Comment:
This does not belong in `config` - it belongs in `manager`.

To resolve the import cycle, I would suggest adding a new interface in `desktop`:

```
type ShutdownHandler interface {
	Shutdown(code int)
}
```

`desktop.Start` should be changed to accept a `ShutdownHandler` parameter. `manager` can create a small internal type that fulfils this interface by calling `Shutdown`.

Reply:
I've implemented the suggested changes.